tools/check_output_test.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# Usage: check_output_test.py [path to deno executable]
+import os
+import sys
+import subprocess
+
+root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+tests_path = os.path.join(root_path, "tests")
+
+
+def check_output_test(deno_fn):
+    assert os.path.isfile(deno_fn)
+    outs = [fn for fn in os.listdir(tests_path) if fn.endswith(".out")]
+    assert len(outs) > 1
+    tests = [(os.path.splitext(fn)[0], fn) for fn in outs]
+    for (script, out_fn) in tests:
+        script_abs = os.path.join(tests_path, script)
```

Comment:
This is probably a pre-existing issue -- it seems that `deno tests\bla.js` works but `deno c:\full\path\to\bla.js` crashes.
I suspect the module resolver mistakes `c:\full\path\...` for a relative path.

```
C:\Users\BertBelder\d\deno>out\wd\deno.exe tests\001_hello.js
Hello World

C:\Users\BertBelder\d\deno>out\wd\deno.exe C:\Users\BertBelder\d\\deno\tests\001_hello.js
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', libcore\result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace
```

Stack:

```
<snip>
   9: core::result::Result<std::path::PathBuf, ()>::unwrap<std::path::PathBuf,()>
             at C:\projects\rust\src\libcore\result.rs:782
  10: handlers::resolve_module
             at .\src\handlers.rs:89
  11: handlers::handle_code_fetch
             at .\src\handlers.rs:191
  12: deno_handle_msg_from_js
             at .\src\reply.cc:91
  13: deno::Send
             at .\src\binding.cc:206
<snip>
```